.github/workflows/push_tests.yml

```diff
@@ -1,9 +1,13 @@
-name: Slow tests on main
```

Comment:
Great let's revert running this workflow on the PR tests and then we can merge :-)